### ISSUE TITLE
Card background colour doesnt update

### ISSUE THREAD

**Issue by joehill1810** [lukin/keywind#68]
Please tell me if I'm not doing something right, but I'm trying to update the main card colour from white (bg-white) to a different colour (emerald in this case), but doing this just makes the card invisible.

<#macro kw content="" footer="" header="">
  <div class="bg-emerald-800 p-8 rounded-lg space-y-6">
    <#if header?has_content>
      <div class="space-y-4">
        ${header}
      </div>
    </#if>
    <#if content?has_content>
      <div class="space-y-4">
        ${content}
      </div>
    </#if>
    <#if footer?has_content>
      <div class="space-y-4">
        ${footer}
      </div>
    </#if>
  </div>
</#macro>

<img width="1800" alt="Screenshot 2023-10-08 at 13 58 53" src="https://github.com/lukin/keywind/assets/31496862/1c2a16c4-c3d7-4a17-90d7-c0fb4966e70b">

**Comment by paulwer** [comment]
have you issued the build command after changing this and before deployment?
it sounds like, the used tailwind is not available within production.

## Build

When you're ready to deploy your own theme, run the build command to generate a static production build.

```bash
pnpm install
pnpm build
```

To deploy a theme as an archive, create a JAR archive with the theme resources.

```bash
pnpm build:jar
```

**Comment by joehill1810** [comment]
> have you issued the build command after changing this and before deployment? it sounds like, the used tailwind is not available within production packed css.
> 
> Build - Readme.md

Yep! Editied card.ftl with the update, ran the build command and moved the folder into the appropriate themes folder.

**Comment by paulwer** [comment]
I wasn't able to reprduce. I did the following:
1. change bg-white to bg-emerald-800
2. run command: npm run build && npm run build:jar (after the first command => the styles.css contains bg-emerald-800 as class)
3. deploy via docker

result in browser:
![Unbenannt](https://github.com/lukin/keywind/assets/52678724/ae835674-2de0-4dbb-b7a1-d2494f7c95d0)
